test/src/e2e_vm_tests/test_programs/token_ops_test/src/main.sw

```diff
@@ -23,9 +23,11 @@ fn main() -> bool {
     };
 
     // the deployed fuel_coin Contract_Id:
-    let fuelcoin_id = ~ContractId::from(0x49050d88db785fdf3b3fd35df44fbe644b9b5273ae97d9c7c19cc3e4aa361eaf);
+    let fuelcoin_id = ~ContractId::from(0xbaf516b8b50c51953a6d2f20ec1f5ff09ced2b80a9c72ea376aaf23dd6af467e);
+    let balance_test_id = ~ContractId::from(0x27eb552a9458aec1db874930ae86fe91df49b4e0c221e08f7ffcf3fadadee0a3);
+
     // todo: use correct type ContractId
```

Comment:
Can this todo be removed now?

Reply:
Not, See: https://github.com/FuelLabs/sway/issues/577